Templates/HttpTriggerWithParameters-CSharp/run.csx

```diff
@@ -0,0 +1,7 @@
+using System.Net;
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req,string name, TraceWriter log)
```

Comment:
Space between parameters

Add a comment explaining where "name" is coming from (via parameters)